system-addon/data/content/assets/glyph-info-16.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path fill="context-fill" d="M8 1a7 7 0 1 0 7 7 7.008 7.008 0 0 0-7-7zm0 13a6 6 0 1 1 6-6 6.007 6.007 0 0 1-6 6zm0-7a1 1 0 0 0-1 1v3a1 1 0 1 0 2 0V8a1 1 0 0 0-1-1zm0-3.188A1.188 1.188 0 1 0 9.188 5 1.188 1.188 0 0 0 8 3.812z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 16a8 8 0 1 1 8-8 8.009 8.009 0 0 1-8 8zM8 2a6 6 0 1 0 6 6 6.006 6.006 0 0 0-6-6z"/><path d="M8 7a1 1 0 0 0-1 1v3a1 1 0 0 0 2 0V8a1 1 0 0 0-1-1z"/><circle cx="8" cy="5" r="1.188"/></svg>
```

Comment:
Oh I forgot to mention this, but it's easier to tell now via this diff. It's missing fill="context-fill"

Reply:
ah, and only that one.

Reply:
`fill: red !important` ;)
<img width="191" alt="image" src="https://user-images.githubusercontent.com/438537/36576380-34c5fec8-1805-11e8-971b-397ff7dad923.png">

Reply:
I suppose this would have missed the white fill too, but that's harder to glance quickly ;)

<img width="174" alt="image" src="https://user-images.githubusercontent.com/438537/36576411-59cdf8e2-1805-11e8-915e-fb366a52649d.png">